src/Main.elm

```diff
@@ -25,40 +143,939 @@ init =
+                    [ Html.span [ Attr.class "loader" ] []
+                    ]
+
+            Error error ->
+                Html.div [ Attr.class "flex items-center z-12 bg-red-200 mt-10 border px-3 py-2 mx-4 rounded border-red-400 text-2xl text-red-400 " ]
+                    [ Html.span [ Attr.class "w-[30px] flex mr-2" ] [ Icons.cautionIcon ]
+                    , Html.p [] [ text error ]
+                    ]
+
+            Success userGroup ->
+                Html.div [ Attr.class "flex flex-col mx-4 md:mx-0 mt-16 mb-48" ]
+                    [ Html.div [ Attr.class "mx-4 md:mx-0" ]
+                        [ Html.div [ Attr.class "flex w-[140px] h-[30px]" ]
+                            [ Html.img [ Attr.src "logo.png" ] []
+                            ]
+                        , Html.h1 [ Attr.class "text-3xl  md:text-5xl mb-10" ] [ text "User Group Settings Form" ]
+                        ]
+                    , if model.isFormSent then
+                        Html.div [ Attr.class "flex mb-4 text-green-400 rounded border border-green-400 bg-green-200 px-3 py-2" ]
+                            [ Html.span [ Attr.class "mr-2 flex w-[20px]" ] [ Icons.checkIcon ]
+                            , Html.p [] [ text "Success ! Form has been sent !" ]
+                            ]
+
+                      else
+                        text ""
+                    , Html.div [ Attr.class "" ]
+                        [ Html.div [ Attr.class "flex flex-col" ]
+                            [ Html.div [ Attr.class "" ]
+                                [ viewTabs model.tabActive ]
+                            , Html.div
+                                [ Attr.class "bg-blue-100 p-4 mb-10 overflow-hidden" ]
+                                [ Html.div [ Attr.class "mb-4" ] (model.errors |> Set.toList |> List.map (\error -> viewErrorMessage error))
+                                , case model.tabActive of
+                                    SettingsTab ->
+                                        viewSettings model.formData userGroup.settings.isInherited model.showTooltip
+
+                                    DetailsTab ->
+                                        viewContactDetails model.formData { isInherited = userGroup.contactDetails.isInherited, prefMethod = model.formData.prefMethod } model.toggleDropdown
+
+                                    TagsTab ->
+                                        viewTags model.formData.tags model.formData.inputFields
+                                ]
+                            ]
+                        , Html.button [ onClick <| SubmitForm { isSettingsInherited = userGroup.settings.isInherited, isDetailsInherited = userGroup.contactDetails.isInherited }, Attr.class "flex font-semi-bold px-5 py-2 rounded-full bg-blue-400 hover:bg-blue-300 active:bg-blue-500 easy-in-out transition-all text-white" ] [ Html.span [ Attr.class "mr-2 flex w-[20px]" ] [ Icons.checkIcon ], text "Submit" ]
```

Comment:
NIT: This is too long for my taste it makes things hard to read for me.

Reply:
Would it makes sense to "break" it in next line and let elm format do the job ? Or is it more in terms of a size of a record + gigantic chained tailwind class ?